h/api/models/elastic.py

```diff
@@ -58,7 +64,28 @@ def target_links(self):
     def document(self):
         return self.get("document", {})
 
+    def __acl__(self):
```

Comment:
At some point, yes. Don't worry about it for now. We're going to need to go through and do an audit of all the properties used by the web application and replicate them on the SQLA model at some point.